kafka/kafka-topic.yaml

```diff
@@ -1,8 +1,10 @@
 apiVersion: kafka.strimzi.io/v1beta2
 kind: KafkaTopic
 metadata:
-  name: ztf-stream-sim
+  name: watchdog
```

Comment:
This is a new topic name? If so, is this supposed to be changed in the fink-broker side as well to be consistent?